public/javascripts/mce_editor.js

```diff
@@ -67,25 +67,28 @@ tinyMCE.init({
   // Override the list of targets provided in the link plugin. We do not allow the target attribute, so we want an empty list.
   target_list: [
     { title: 'None', value: '' }  
-  ]
+  ],
+
+  // Options to display in the context (right-click) menu.
+  contextmenu: 'paste | link | hr'
```

Comment:
The default configuration is `link image imagetools table spellchecker`. We should keep `image` (it only shows up when you right click on an image).

Reply:
Ah, I thought it was lying because I wasn't seeing that. I will add it!